File: src/2652-Sum-Multiples/sum_multiples.py
class Solution:
    def sumOfMultiples(self, n: int) -> int:
        ans: int = 0

        for i in range(1, n + 1):
            if i % 3 == 0 or i % 5 == 0 or i % 7 == 0:
                print(i)
                ans += i

        return ans

if __name__ == "__main__":
    n: int = 7
    
    solution = Solution()

    print(solution.sumOfMultiples(n))

File: src/2652-Sum-Multiples/sum_multiples.cpp
#include <iostream>

class Solution {
public:
    int sumOfMultiples(int n) {
        int ans = 0;

        for (int i = 0; i <= n; i++) {
            if (i % 3 == 0 || i % 5 == 0 || i % 7 == 0) {
                ans += i;
            }
        }

        return ans;
    }
};

int main() {
    int n = 4;    
    Solution solution;

    std::cout << solution.sumOfMultiples(n);

    return 0;
}             